examples/chat/README.md

```diff
@@ -15,8 +15,14 @@ as bearer tokens:
 
 ## Command Line Operation
 
-The first client to run typically creates the Oak Application and a first chat
-room inside it:
+Initially, the chat example needs to be built:
+
+```bash
+./scripts/build_example -e chat
+```
+
```

Comment:
While you're here, maybe add something about starting a runtime instance with the chat application in it (i.e. `scripts/run_server -e chat`) ?

Reply:
Happy to do that! I was a bit puzzled by line 24 which ("After this, one can run the first client, which typically creates the Oak Application"). 

It is indeed the case that the server needs to be run separately though. 

Out of curiosity, what's the difference of the "Oak Application" and server?

Reply:
That's a historical reference that didn't get updated &ndash; the clients used to send the compiled Wasm code for an Oak Application to the runtime server, which would run it.  That didn't really make sense for any real deployment, so it was changed so that:
 - The "server" is the Oak Runtime, which is a binary that's capable of running any Oak Application in its Wasm engine.
 - The Oak Application is built as a pile of compiled Wasm code, which gets passed to the Runtime as a start-of-day command line option, and which it runs in its Wasm engine.
 - The client connects to the Oak Runtime over gRPC, and the Runtime forwards any gRPC requests that arrive into the Oak Application code running inside the Wasm engine.

The word "server" does get used in other contexts,though &ndash; for example, the Oak Application code (running inside Wasm) acts as a gRPC "server", in that it receives gRPC method invocations in the form of gRPC requests that it responds to with gRPC responses.

Reply:
Oh that makes sense, thank you for explaining. :) 

Based on that I'd propose changing line 31 to say "After this, one can run the first client, which connects to the Oak Application and opens a first chat room inside it:"

Does that sound good?

Reply:
SGTM

Reply:
You sure like variations on LGTM 😁